pkgs/development/libraries/gdk-pixbuf/default.nix

```diff
@@ -42,6 +42,8 @@ stdenv.mkDerivation rec {
   patches = [
     # Move installed tests to a separate output
     ./installed-tests-path.patch
+    # Allow for multiple loader.cache files specified in $GDK_PIXBUF_MODULE_FILE, delimited by ":"
+    ./multiple-module-files.patch
```

Comment:
Could this be submitted upstream? That way we could get some upstream reviews for it.

Reply:
We can try, but I doubt upstream will be very happy about this. The environment variable is mostly meant for development and what we are doing is an extremely dirty hack.